GNUmakefile

```diff
@@ -0,0 +1,61 @@
+# If you see pwd_unknown showing up check permission
+PWD ?= pwd_unknown
+
+# DATABASE DEPLOY FOLDER CONFIG - default ./data
+ifeq ($(data),)
+DATA := data
+export DATA
+else
+DATA := $(data)
+export DATA
+endif
+
+help:
+	@echo ''
+	@echo ''
+	@echo '	Usage: make [COMMAND]'
+	@echo ''
+	@echo '		make init'
+	@echo '		make all'
+	@echo '		make torproxy'
+	@echo ''
+	@echo '	Docker Commands:'
+	@echo ''
+	@echo '		make d-ps'
+	@echo '		make d-images'
+	@echo ''
+	@echo ''
+
+.PHONY: d-ps
+d-ps:
+	@echo ''
+	bash -c 'docker ps'
+	@echo ''
+.PHONY: d-images
+d-images:
+	@echo ''
+	bash -c 'docker images'
+	@echo ''
+
+.PHONY: init
+init:
+	@echo ''
+	mkdir -p   $(DATA) $(DATA)/mysql $(DATA)/mysql/db-scripts $(DATA)/mysql/data 
+	install -v mariadb-structure.sql $(DATA)/mysql/db-scripts
+	@echo ''
+	@echo 'REF: https://github.com/mempool/mempool/blob/master/docker/README.md'
+	cat docker/docker-compose.yml > docker-compose.yml
+
+.PHONY: all
+all: init
+	@echo 'all'
+	docker-compose up --force-recreate --always-recreate-deps
+	@echo ''
+#######################
+.PHONY: torproxy
+torproxy:
+	@echo 'torproxy'
+#REF: https://hub.docker.com/r/dperson/torproxy
+	bash -c 'docker run -it -p 8118:8118 -p 9050:9050 -p 9051:9051 -d dperson/torproxy'
```

Comment:
Why is this needed? It seems to be not used for anything, and it adds a trusted third party dependency.

Reply:
I will take it out :)